lib/src/logic.dart

```diff
@@ -592,4 +592,25 @@ class Logic {
       getRange(0, startIndex),
     ].swizzle();
   }
+
+  /// Returns true if the [Logic] calling this function is in [List] else false.
+  ///
+  /// The list can be [Logic] or [int] or [bool] or [BigInt] or mix of all
+  /// ([dynamic]) supplied
+  ///
+  bool isIn(List<dynamic> list) {
```

Comment:
Since this operates on a `Logic`, we want this function to return a 1-bit `Logic` rather than a `bool`.